scripts/cpu_info.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+get_percent()
+{
+	arg=$1
+	case "$arg" in
+		Linux)
+			percent=$(top -bn1 | grep "Cpu(s)" | sed "s/.*, *\([0-9.]*\)%* id.*/\1/" | awk '{print 100 - $1"%"}')
+			echo $percent
+		;;
+		
+		Mac)
+			percent=$(ps -A -o %cpu | awk '{s+=$1} END {print s "%"}')
+			echo $percent
+		;;
+
+		Windows)
+			# TODO - windows compatability
+		;;
+	esac
+}
+check_os()
```

Comment:
Do we need to make checking the OS a seperate function? Checking the OS in get_percent() would be cleaner and more consistent with code previous code written which needs to check for OS in battery.sh and network.sh

Reply:
No, it doesn't need to be, I fixed that. Sorry.

Reply:
oh crap, my branch wasnt updated, let me fix that